### DISCUSSION TITLE
#100DaysOfGatsby - Challenge 2

### DISCUSSION THREAD

**Discussion by Aereli** [gatsbyjs/gatsby#29310] [Community]
Hi all, Just wondering if anybody has finish or worked on [challenge 2](https://www.gatsbyjs.com/blog/tags/100-days-of-gatsby/)
I dislike/like how far the boundaries of gatsby are being pushed here.
This is mine so far.. 
https://100daysofgatsby2021.netlify.app/

**Comment by jurek2006** [comment]
Hi, I have been working on challenge 2.

My project is a big mess now. I have been playing with the features mentioned in the challenge. But I just get acquainted a bit with one after one (chakra-ui, formium, gatsby-plugin-image, etc.) and I hope I will have time to join it nicely together. (Like styling formium with chakra, building consistent navBar with chakra).
I like that it shows unknown (to me) fields and solutions. But with some things, I find it hard to find good information, documentation - like how to use gatsby-plugin-image for images from Contentful [I know, it's just beta. So I used the old gatsby-image] or how to style nicely formium form with chakra. So I experiment with it.

**Reply by rodneylab** [reply]
Hi Jurek,
Obinna posted something on gatsby-plugin-image on his Twitter feed, dunno if you saw it.  Might be useful [https://twitter.com/Obinnaspeaks/status/1361634556785352704/photo/1](https://twitter.com/Obinnaspeaks/status/1361634556785352704/photo/1).
Rodney

**Comment by orgabor** [comment]
This is mine from [challenge 2](https://www.gatsbyjs.com/blog/tags/100-days-of-gatsby/) : https://audioc0reorgabor.gtsb.io/

**Comment by tdudkowski** [comment]
https://challengegatsby100.gtsb.io/
I'm not happy with this one, but still before deadline. It's before 17.00, yep?
...and awaiting orders

**Comment by Kevpedia** [comment]
I got stuck on preact... normally just a plug-and-play gatsby plugin, but can't seem to get it to work in this project... Hoped to see how they did it in the live-stream but then they _barely_ made it past installing chakra and making a contact form

**Reply by rodneylab** [reply]
Hi Kevin, what happens with preact? For me I just installed it and it worked.

**Reply by Kevpedia** [reply]
I get this message when I try to run gatsby develop or build+serve. I have tried running gatsby clean and that didn't fix it...
<img width="1439" alt="image" src="https://user-images.githubusercontent.com/16330251/108600707-b0807880-7366-11eb-894c-fc17a3cc84cf.png">

The site won't load until I click 'stop' and when it does I get this loading indicator that won't go away:
<img width="260" alt="image" src="https://user-images.githubusercontent.com/16330251/108600734-d3129180-7366-11eb-80de-e5479811cfe7.png">

**Reply by Kevpedia** [reply]
I see this message in the terminal where I'm running gatsby develop:
<img width="1018" alt="image" src="https://user-images.githubusercontent.com/16330251/108601025-5c769380-7368-11eb-8a81-008acb382e2e.png">

**Reply by Kevpedia** [reply]
I also tried deleting node_modules folder, running gatsby clean again, and then running build; but no luck

**Reply by Kevpedia** [reply]
https://github.com/Kevpedia/audio-c0re/tree/preact

**Reply by rodneylab** [reply]
Hi Kevin, sorry I took so long to come back. Took a look at your code and didn't see anything that stood out, wanted to try running in locally but didn't get a chance.  Anyway I saw something simiilar on another site and it disappeared when I upgraded it to Gatsby 3.  I would try that if you have the time.  Here's a link on migrating https://www.gatsbyjs.com/docs/reference/release-notes/migrating-from-v2-to-v3/

**Reply by rodneylab** [reply]
I get this warning in v3

warn Plugin @chakra-ui/gatsby-plugin is not compatible with your gatsby version
3.0.0 - It requires gatsby@^2.24.85

but everything seems to work

**Comment by rodneylab** [comment]
Here's what I have for Challenge 2.  I have open-sourced my code. Is this helpful for anyone who is stuck?  Let me know if you have any questions. Rodney

Live site: <a aria-label="Open Rodney Lab Gatsby 100 Days of Code Challenge Site" href="https://rodneylab100daysofgatsbymain.gtsb.io/home" target="blank" rel="noopener noreferrer">https://rodneylab100daysofgatsbymain.gtsb.io/home</a>
Code: <a aria-label="Open Rodney Lab Gatsby 100 Days of Code Challenge Site Code" href="https://github.com/rodneylab/rodneylab-100-days-of-gatsby" target="blank" rel="noopener noreferrer">https://github.com/rodneylab/rodneylab-100-days-of-gatsby</a>

**Comment by humbl3man** [comment]
here is mine: https://audiocore-25.netlify.app/
